include/openenclave/corelibc/sys/bits/syscall_arm.h

```diff
@@ -0,0 +1,359 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#define OE_SYS_restart_syscall 0
```

Comment:
No, because this file is private and only included by **<syscall.h>**. The guard would not harm anything, but not needed. MUSL and other libc's use the same pattern (single inclusion case).